src/compiler/scala/tools/nsc/backend/jvm/PostProcessor.scala

```diff
@@ -63,7 +65,10 @@ abstract class PostProcessor extends PerRunInit {
         backendReporting.error(NoPosition,
           s"Could not write class ${internalName} because it exceeds JVM code size limits. ${e.getMessage}")
         null
+      case ex: InterruptedException => throw ex
       case ex: Throwable =>
+        // TODO hide this stack trace behind -Ydebug?
+        // TODO fail fast rather than continuing to write the rest of the class files?
         ex.printStackTrace()
```

Comment:
Same question here

Reply:
Agree to guard with `if (settings.debug)`. I also agree to fail fast instead of continuing with writing other classfiles.